src/vcpkg/configuration.cpp

```diff
@@ -856,32 +856,22 @@ namespace vcpkg
     {
         Json::Reader reader(origin);
         auto maybe_configuration = ConfigurationDeserializer::instance.visit(reader, obj);
-        bool has_warnings = !reader.warnings().empty();
-        bool has_errors = !reader.errors().empty();
-        if (has_warnings || has_errors)
+        if (!reader.messages().good())
         {
-            if (has_errors)
+            if (reader.messages().any_errors())
             {
-                messageSink.println(Color::error, msgFailedToParseConfig, msg::path = origin);
-            }
-            else
-            {
-                messageSink.println(Color::warning, msgWarnOnParseConfig, msg::path = origin);
```

Comment:
Technically I removed this message but I can't imagine this adds value over the existing warning text.